layouts/partials/footer.html

```diff
@@ -0,0 +1,83 @@
+<footer id="site-footer" class="py-10 print:hidden">
```

Comment:
What is the reason to add the `funded by the European<br>Commission` here and not in the `extend-footer.html`?

Reply:
the site had two footer parts: the footer and extended footer (the one on top of the other). It seemed a little bit weird. If you don't like it like this we can revert it to its original state.

Reply:
It's ok - I think now that we have settled on the theme it will work well. We should just look out for changes to the theme over time that we want to have.